src/Todo.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+function Todo({ id, content, onClick }) {
+  return (
+    <li>
+      {content}
+      <button type="button" onClick={() => onClick(id)}>완료</button>
```

Comment:
```suggestion
      <button type="button" onClick={() => onClickDelete(id)}>완료</button>
```

이런 경우는 property인 onClick과  구분을 위해 명확하게 써주는 편이 좋아요.

Reply:
`onClickDelete`로 변경하고 기존 함수도 `handleClickDeleteTodo`로 변경하였습니다.